### ISSUE TITLE
How to use `vertical_tabs.css`?

### ISSUE THREAD

**Issue by Marin-Kitagawa** [MrOtherGuy/firefox-csshacks#363]
I have used the `vertical_tabs.css` along with `window_control_placeholder_support.css` and `hide_tabs_toolbar.css`. No tabs appear on the left sidebar. I even tried setting `sidebar.revamp` to `true` in `about:config`. It just appears blank. Could you please suggest how to use it? 

THe following is the content of my `userChrome.css`

```css
@import url(chrome/tab_close_button_always_on_hover.css);
@import url(chrome/minimal_popup_scrollbars.css);
@import url(chrome/button_effect_icon_glow.css);
@import url(chrome/urlbar_centered_text.css);
@import url(chrome/window_control_placeholder_support.css);
@import url(chrome/hide_tabs_toolbar.css);
@import url(chrome/round_ui_items.css);
@import url(chrome/tabs_animated_gradient_border.css);
@import url(chrome/autohide_tabs_toolbar.css);
@import url(chrome/curved_tabs.css);
```

Below is my `userContent.css`

```css
@import url(content/css_scrollbar_width_color.css);
@import url(content/transparent_reader_toolbar.css);
```

**Comment by MrOtherGuy** [comment]
hide_tabs_toolbar.css will hide the tabs, so remove that if you intend to show them vertically.

I'm not sure about all of these, but I can tell you that tabs_animated_gradient_border.css nor autohide_tabs_toolbar.css are not going to work at all with vertical_tabs.css, curved_tabs.css probably won't either.

**Comment by Marin-Kitagawa** [comment]
Oh ok thanks. I will try it

**Comment by MrOtherGuy** [comment]
Yes indeed, that style makes the normal tabs toolbar appear vertically. Which is also why certain features will not work such as ability to reorder tabs because that function depends on tabs being laid out horizontally.
